apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerException.cs

```diff
@@ -84,6 +86,25 @@ public bool IsRetryable
             }
         }
 
+        /// <summary>
+        /// For retryable <see cref="SpannerException"/>, this value represents
+        /// the recommended minimum retry delay. It might or might not
+        /// be present, even if this exeption is retryable.
+        /// </summary>
+        public TimeSpan? RecommendedRetryDelay { get; }
+
+        /// <summary>
+        /// For retrying aborted commits, we need to diferentiate whether the operation
+        /// was aborted due to a session expiry. If the session expired we cannot
```

Comment:
Can we not retry, or can we just not retry with the same session? I thought it was just more efficient if we reuse the sesssion.

Reply:
Okay, I think I agree :)